src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java

```diff
@@ -200,7 +201,11 @@ public void postNotifyUsernameByUsername(String userName, String lang,
       okapiHeaders.get(RestVerticle.OKAPI_HEADER_TENANT));
     String okapiURL = okapiHeaders.get("X-Okapi-Url");
     HttpClientInterface client = getHttpClient(okapiURL, tenantId);
-    String url = "/users?query=username=" + userName;
+    String cql = userName == null ? "" : userName;
+    // mask special CQL characters: \ " * ? ^
+    cql.replace("\\", "\\\\").replace("\"", "\\\"").replace("*", "\\*").replace("?", "\\?").replace("^", "\\^");
```

Comment:
Do we have a RMB util method for this? If not, it would be nice to have one.

Reply:
Good suggestion, done: #776